lib/generators/spree_piwik/install/install_generator.rb

```diff
@@ -6,12 +6,17 @@ class InstallGenerator < Rails::Generators::Base
       class_option :auto_run_migrations, :type => :boolean, :default => false
 
       def add_javascripts
+        append_file 'vendor/assets/javascripts/spree/frontend/all.js', "//= require spree/frontend/spree_piwik\n"
+        append_file 'vendor/assets/javascripts/spree/backend/all.js', "//= require spree/backend/spree_piwik\n"
```

Comment:
Line is too long. [111/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.